language/tools/move-cli/src/package/cli.rs

```diff
@@ -6,11 +6,18 @@ use std::{
     fmt::Display,
     fs::{create_dir_all, read_to_string},
     io::Write,
-    os::unix::prelude::ExitStatusExt,
     path::{Path, PathBuf},
     process::ExitStatus,
 };
 
+// in windows
+#[cfg(target_os = "windows")]
+use std::os::windows::process::ExitStatusExt;
+
+// in unix
+#[cfg(not(target_os = "windows"))]
```

Comment:
I assume target_os should be unix here?

Reply:
I think it is better to do effectively if-then-else here (as is done), which the not(windows) does here.  Otherwise if not(windows) and not(linux), you have no import for this at all.